.gitignore

```diff
@@ -1,3 +1,4 @@
 /target
 **/*.rs.bk
 Cargo.lock
+*id_rsa*
```

Comment:
Why don't you read and write the example data inside the `target/` directory.

Reply:
Good point, I'll change the default file location.